mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -313,6 +313,10 @@ includes an up-to-date reference of supported versions.
 
 * `root_password` - (Optional) Initial root password. Can be updated. Required for MS SQL Server.
 
+* `root_password_wo` - (Optional) Initial root password. Can be updated. Required for MS SQL Server. For more info see [updating write-only arguments](/docs/providers/google/guides/using_write_only_arguments.html#updating-write-only-arguments)
+
+* `root_password_wo_version` - (Optional) Triggers update of root password write-only. For more info see [updating write-only arguments](/docs/providers/google/guides/using_write_only_arguments.html#updating-write-only-arguments)
+
```

Comment:
Can you also add a note for the write only field to the top of this doc, similar to [this note](https://github.com/hashicorp/terraform-provider-google/blob/main/website/docs/r/secret_manager_secret_version.html.markdown?plain=1#L43)? Thanks.